easybuild/easyconfigs/m/MView/MView-1.67-foss-2022a-Perl-5.34.1.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = "Tarball"
+
+name = 'MView'
+version = "1.67"
+versionsuffix = '-Perl-%(perlver)s'
```

Comment:
no need for a `versionsuffix` here, since there is only one `Perl` version in the toolchain

```suggestion
```